services/SettingsFileService.py

```diff
@@ -0,0 +1,95 @@
+import json
+import logging
+import os
+import shutil
+
+import jsonmerge
+
+from services.utils import PathUtils
+
+
+class SettingsFileService:
+    _default_file_name: str = 'default.json'
+    _definitive_file_name: str = 'settings.json'
+    _temporary_file_name: str = 'settings_temp.wl'
+
+    def resetTemporaryFile(self):
+        try:
+            if os.path.exists(PathUtils.settings(self._temporary_file_name)):
+                os.remove(PathUtils.settings(self._temporary_file_name))
+
+                shutil.copy(
+                    PathUtils.settings(self._definitive_file_name), PathUtils.settings(self._temporary_file_name)
+                )
+        except Exception as err:
+            logging.error('It was not possible to delete the temporary file.', err)
+
+    def loadSettingsFile(self, file_type: int):
+        if file_type == 0:
+            file_name = self._default_file_name
+        elif file_type == 1:
```

Comment:
Would be better to allocate this value into a variable that can describe what this value represents.